### PR TITLE
Skip osx-arm64 build for Python 3.8

### PR DESCRIPTION
**Pull request by ccordoba12**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

- That's because dependencies can't be met for that version.
- See the discussion in https://github.com/conda-forge/applaunchservices-feedstock/pull/6.

### CODE REVIEW

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +28,7 @@ requirements:
     - setuptools
   run:
     - python
-    - applaunchservices >=0.3.0  # [osx]
+    - applaunchservices >=0.3.0  # [osx and (x86_64 or (arm64 and py >= 39))]
```
Comment (mbargull) [inline recipe/meta.yaml:31]: This can stay as was since the `arm64 and py < 39` are skipped, i.e., everything after `osx` will be `true`.

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

**Pull request merged**